### ISSUE TITLE
3rd party funding

### ISSUE THREAD

**Issue by jfromaniello** [uport-project/ethr-did#98]
The docs for `ethr-did-registry` mention this:

> Since each Ethereum transaction has to be funded, there is a growing trend of on-chain transactions that are authenticated via an externally created signature and not by the actual transaction originator. This allows 3rd party funding services or receiver pays without any fundamental changes to the underlying Ethereum architecture.

I assume this is done by using the `*Signed` version of the methods like `setAttributeSigned`, but I couldn't find support for this in this library nor in the `ethr-did-resolver`.

- Is there any example about this?
- What would it take to implement this feature?

Thanks!

**Comment by mirceanis** [comment]
The only samples of this in action are in the registry test suite: https://github.com/uport-project/ethr-did-registry/blob/bd6f1b78797baccb3b9ab1019fd61cf9890aa304/test/registry.test.ts#L587

Do you want to implement that kind of functionality here?

**Comment by jfromaniello** [comment]
Thank you @mirceanis , I also saw this test case.

First of all, I wanted to ask here because I wasn't sure "meta transactions" weren't implemented already, for example in ethers.

I am not sure if it should be here. I am just trying out did implementations and the documentation for did:ethr points to this library. I was able to add a public new public key and change the services in the did document, but I was wondering how for instance a wallet implementor/hub could take care of the fees and that's where I came across that section of the documentation.

If you think is a good idea to have it here I could give it a try to port the test case of the registry.

Thanks in advance
